src/main/java/no/nav/svangerskapspenger/tjeneste/opprettperioder/UttaksperioderTjeneste.java

```diff
@@ -105,6 +105,9 @@ private BigDecimal finnUtbetalingsgrad(Søknad søknad, Tilrettelegging tilrette
         if (kryss.equals(TilretteleggingKryss.A)) {
             return BigDecimal.ZERO;
         } else if (kryss.equals(TilretteleggingKryss.B)) {
+            if (tilrettelegging.getOverstyrtUtbetalingsgrad().compareTo(BigDecimal.ZERO) > 0) {
```

Comment:
Tenk på konvensjon for å sjekke overstyring - null vs det å ha overstyrt til utbetalingsgrad 0%

Reply:
godt poeng!

Reply:
har fikset i ny commit